website/docs/ide-config.md

```diff
@@ -0,0 +1,62 @@
+<h2 id="vscode">VS Code</h2>
+
+> VS Code Extensions
+
+- ESLint
+  - id: "dbaeumer.vscode-eslint"
+  - recommend reason: "Integrates ESLint JavaScript into VS Code."
+- Stylelint
+  - id: "stylelint.vscode-stylelint"
+  - recommend reason: "Official Stylelint extension for Visual Studio Code"
+- GitLens
+  - id: "eamodio.gitlens"
+  - recommend reason: "It helps you to visualize code authorship at a glance via Git blame annotations and CodeLens"
+- markdownlint
+  - id: "DavidAnson.vscode-markdownlint"
+  - recommend reason: "Markdown linting and style checking for Visual Studio Code"
+
+> VS Code Settings
```

Comment:
The other thing is, how do we make sure that the configuration in the `.vscode` folder and here can be maintained at the same time? I would suggest adding a "Tip" here